### ISSUE TITLE
Turn off Logging 

### ISSUE THREAD

**Issue by Nath5** [vendelieu/telegram-bot#149]
I am trying to turn off all the logging outside of errors but despite toggling the logging settings I am still seeing a bunch of INFO level lines in my log. Is there something else that needs to be done to turn off these log messages?

Running version: 5.4.0 
Kotlin Version: 1.9.23

Log Messages
```
2024-05-22 15:40:26,675 INFO  [DefaultDispatcher-worker-3] e.vendeli.tgbot.utils.LogbackLogger (26): Handling update #122579515
2024-05-22 15:40:26,677 INFO  [DefaultDispatcher-worker-3] e.vendeli.tgbot.utils.LogbackLogger (26): Update #122579515 processed in functional mode with whenNotHandled action.
2024-05-22 15:40:26,677 INFO  [DefaultDispatcher-worker-3] e.vendeli.tgbot.utils.LogbackLogger (26): Number of affected functional actions - 1.
```

Bot Setup
```
    val tBot = TelegramBot(settings.telegramSettings.token){
        logging {
            botLogLevel = LogLvl.ERROR
            httpLogLevel = HttpLogLevel.NONE
        }
    }
```

Logging Dependencies
```
    // Logging //
    implementation("org.slf4j:slf4j-api:2.0.13")
    implementation("ch.qos.logback:logback-core:1.5.6")
    implementation("ch.qos.logback:logback-classic:1.5.6")
```

Thanks for any help you can offer.

**Comment by vendelieu** [comment]
Did you tried without declaring logging dependencies, library have logback as transitive dependency, I may be wrong but I've encountered something similar when I had them :/.
A bit later I'll look into it :)

**Comment by vendelieu** [comment]
I found why this problem occured, thank you for your report!
For now to change logging you can place `logback.xml` file in your `resources` folder with such content:
```xml
<configuration>
    <logger name="TgUpdateHandler" level="ERROR"/>
</configuration>
```

**Comment by Nath5** [comment]
I will give that a try. Thank you for digging in.

**Comment by vendelieu** [comment]
should be fixed in 5.5.0
